services/order.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None,
+) -> None:
+
+    the_user = get_user_model().objects.get(username=username)
```

Comment:
The prefix `the_` is not necessary. Simply using `user` would be more conventional and clear

Reply:
if you want to make it more clear, you can use `order_date` instead of `date`